bzt/modules/aggregator.py

```diff
@@ -59,6 +59,9 @@ def get_percentiles_dict(self, percentiles):
     def get_counts(self):
         return self.histogram.get_value_counts()
 
+    def get_stdev(self):
+        return self.histogram.get_stddev()
```

Comment:
This will cause iteration over histogram. Can we avoid this? Old solution was getting stdev and percentiles together, to avoid double pass...

Reply:
Don't forget the benefit of performance they bring. I'm sure they fit fine into our usage, assuming we're not shy to adapt.